.github/workflows/deploy.yml

```
@@ -0,0 +1,41 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches: ['main']
```

let's use this syntax as per the [documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs)

```suggestion
    branches:
      - main
```